Add markdown format

CarlesCG opened the pull request:
- Add markdown format to README.md
- Also add the code to install from your repo on GitHub

CarlesCG committed:
Add markdown format

CarlesCG committed:
Import pipe operator %>% and translate example 1

jto888 commented:
Well Carles, its good to 'see' you again.  Getting familiar with 
markdown is on my todo list, unfortunately it gets buried deeper each 
time I look.  I was quite interested in removing the extra line spaces 
between lines of code, particularly for the pwr example. This example is 
also developed in the 4th 'chapter' of the user_tutorial, so I was even 
considering dropping it from this readme.

In your version the visualization is broken.  I was really thinking that 
at that point readers could have generated the visualization of pwr by 
example anyway.

The reference to installation from github is fine.  This is not really 
for those new to R because I would have to discuss the installation of 
devtools and even the fact that you would prefer to be running an R 
session 'as administrator' for a single library.  FaultTree seems to be 
a magnet for new R users.  Next item on my todo was to add reference to 
the user_tutorial in this readme. So far, the repository on R-Forge is 
gaining more initial attention than the github.

The Example Scripts title should be below the installation from github 
section.

As a minimum I plan to reformat the user_tutorial using the Springer 
Word template.  It would be great to learn enough LaTeX to use their TeX 
template, but that is further down on the todo list.

As is, the user_tutorial geared for new R users and fault tree learners 
is too large for a vignette and too small for a book.  A vignette for 
those more experienced with both might be a good addition.  I have 
resolved to document the entire package as a second section, to flesh 
out an eventual book.

On 8/22/2016 6:56 PM, Carles C.G. wrote:

> - Add markdown format to README.md
> - Also add the code to install from your repo on GitHub
> 
> ---
> 
> ```
>     You can view, comment on, or merge this pull request online at:
> ```
> 
> https://github.com/jto888/FaultTree/pull/5
> 
> ```
>     Commit Summary
> ```
> - Add markdown format
>   
>   ```
>   File Changes
>   ```
> - _M_ README.md
>   https://github.com/jto888/FaultTree/pull/5/files#diff-0 (170)
>   
>   ```
>   Patch Links:
>   ```
> - https://github.com/jto888/FaultTree/pull/5.patch
> - https://github.com/jto888/FaultTree/pull/5.diff
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> https://github.com/jto888/FaultTree/pull/5, or mute the thread 
> https://github.com/notifications/unsubscribe-auth/AIc65LOyIbAFnjd5mij9TL2ikUu9U8Gsks5qiikDgaJpZM4JqY69.

CarlesCG committed:
- Fix typo Example 3
- Move 'Example scripts' after installation
- Remove space between lines of code

CarlesCG commented:
Good indeed! Glad to see that you are active around GitHub.
- In the [new commit](https://github.com/CarlesCG/FaultTree/commit/b54e69ec9dc82722f32c0842c1c6ccb4774261d4): typo `pwr` fix , location of ‘example script’ and the line spacing removed. 
- The Example 3, shows clearly that your package can handle big trees. I quite like it at the Readme where the initial user can see it. 
- I saw the user_tutorial repo and I believe you could benefit from rmarkdown syntax & similar stuff as [bookdown](https://bookdown.org). Independently of what you decide to do afterwards, either to go for vignette, [book](https://bookdown.org/csgillespie/efficientR/programming.html) or [html guide](https://cran.r-project.org/web/packages/tidyr/vignettes/tidy-data.html) you will be able to recycle the text and the format. No need to learn LaTeX.

CarlesCG committed:
- Delete import pipe operator
